src/wp-includes/class-wp-block-template.php

```diff
@@ -100,4 +100,12 @@ class WP_Block_Template {
 	 * @var boolean
 	 */
 	public $has_theme_file;
+
+	/**
+	 * Whether a template is a custom template.
+	 *
+	 * @since 5.8.0
+	 * @var boolean
```

Comment:
```suggestion
	 * @since 5.9.0
	 *
	 * @var boolean
```
nitpick can be changed at commit.